packages/cozy-client/src/helpers.js

```diff
@@ -119,6 +119,24 @@ export const deconstructCozyWebLinkWithSlug = (
   }
 }
 
+/**
+ * Deconstruct the given redirect link in order to retrieve slug and hash
+ *
+ * @param {string} redirectLink - redirect link to deconstruct (i.e. 'slug/#/path')
+ * @returns {RedirectLinkData} Deconstructed link
+ */
+
+export const deconstructRedirectLink = redirectLink => {
+  const [slugWithEndingSlash, hash] = redirectLink.split('#')
```

Comment:
Be careful, you can have : 

`slug/path/#/hash`

Reply:
Oh! Like what? `drive/public/#/folder/abcd` ?

Reply:
Yep you don't know what can be there. 

See https://github.com/cozy/cozy-stack/blob/63782e4e45faed57e3fa5fe0ae5c49480a5f6aaf/model/instance/instance.go#L531-L532